.circleci/config.yml

```diff
@@ -222,7 +250,7 @@ workflows:
             - run-tests                            # staging should only deploy if tests pass
             - sam_build                       # deploy needs .aws-sam/build/
           context: [ deployment-staging-election-leaflets ]
-          filters: { branches: { only: [ master ] } }
+          filters: { branches: { only: [ master, upgrades202409-part2 ] } }
```

Comment:
We should remove this branch filter before merging this PR

Reply:
I've added a note in the PR description as a reminder.

Reply:
This is done with https://github.com/DemocracyClub/electionleaflets/pull/255/commits/26b3ee2deeeffdc3e1541fe1634a79fa6f8f6c5a